cabal-install/src/Distribution/Client/Init/Interactive/Command.hs

```diff
@@ -286,7 +286,6 @@ cabalVersionPrompt flags = getCabalVersion flags $ do
 
     displayCabalVersion :: CabalSpecVersion -> String
     displayCabalVersion v = case v of
-      CabalSpecV1_24 -> "1.24  (legacy)"
```

Comment:
After this removal, '+' on the next line doesn't make sense, probably.

Reply:
Good point, How can I cancel checks that are in progress? Under Actions -> workflow I can't seem to find a cancel button.

Reply:
> How can I cancel checks that are in progress?

Don't worry. However, I've just invited you to Triage the repo, so there's a small chance you can see the cancel buttons now (but remember to never mouse over the big red button!).

Reply:
Thank you for that. I believe this solves issue #8680. I also added a changelog file as it was stated in the contribution guide.

Reply:
One thing I was wondering about; I ran the test suite locally on my computer, and I got that a test failed, whereas here everything passes. Is there a known reason for that?

Reply:
I definitely used to have tests darling locally and succeeding in CI. I think one reason may be is some system dependents that may be absent on your machine. It's not nice but it's probably fine.